tensorflow/java/src/main/java/org/tensorflow/Operation.java

```diff
@@ -97,6 +97,30 @@ public Output output(int idx) {
     return new Output(this, idx);
   }
 
+  /**
+   * Returns the size of the given inputs list of Tensors for this operation.
+   *
+   * <p>An Operation has multiple named inputs, each of which contains either
+   * a single tensor or a list of tensors. This method returns the size of
+   * the list of tensors for a specific named input of the operation.The length
+   * of the list should be equal to the number of inputs specified by this
+   * operation's op def.
```

Comment:
I don't believe we explain what an "op def" is in the API documentation, and we don't really need to for most users. Mind rephrasing this to avoid introducing a new term to the reader? Perhaps you can do something very similar to the Javadoc for `outputListLength`? Which I think just means removing this last sentence.